sfklCoding.cpp

```diff
@@ -305,6 +290,37 @@ int ReadHeader(V2_FILEHEADER *FileHeader, BYTE *fbuf, int bufsize)
   return SFARKLIB_SUCCESS;
 }
 
+// =================================================================================
+void InitFilenames(const char *OrigFileName, const char *InFileName, const char *ReqOutFileName)

```

Comment:
`InFileName` is not used here

Reply:
I'm leaving this one as-is for compatibility with my fork...